app/src/main/java/com/keylesspalace/tusky/util/TouchDelegateHelper.kt

```diff
@@ -0,0 +1,61 @@
+@file:JvmName("TouchDelegateHelper")
+
+package com.keylesspalace.tusky.util
+
+import android.graphics.Rect
+import android.view.MotionEvent
+import android.view.TouchDelegate
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.doOnLayout
+import at.connyduck.sparkbutton.helpers.Utils
+
+/**
+ * Ensures the given children of the view will have a touch area of at least the given size, using
+ * a [TouchDelegate].
+ */
+@JvmOverloads
+fun ViewGroup.ensureMinTouchSize(childIds: List<Int>, minSize: Int = Utils.dpToPx(context, 48)) {
```

Comment:
we would you pass list of ids instead of the views themselves? We could avoid expensive `findViewById` again

Reply:
It was an attempt to prevent you passing non-child views but looking back at it not a particularly good one. These days findViewById doesn't really cause a performance problem as if your view hierarchy is deep enough, it'll cause layout slowdowns long before searching gets slow. Don't mind changing it though.